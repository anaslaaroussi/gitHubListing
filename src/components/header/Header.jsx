import React from "react";
import styles from "./styles";

export const Header = () => {
  return <div />;
};
