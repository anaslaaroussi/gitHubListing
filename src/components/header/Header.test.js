import { shallow } from "enzyme";
import React from "react";
import { Header } from "./Header";

// snapshot test

it("expect render Header component", () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
