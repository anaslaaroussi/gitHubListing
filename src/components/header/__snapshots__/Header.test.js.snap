// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render Header component 1`] = `ShallowWrapper {}`;
